src/app.rs

```diff
@@ -565,120 +474,4 @@ impl<Ms, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> App<Ms, Mdl, INodes
             }
         }))
     }
-
-    #[deprecated(
-        since = "0.5.0",
-        note = "Use `builder` with `AppBuilder::{after_mount, before_mount}` instead."
-    )]
-    pub fn build(
-        init: impl FnOnce(Url, &mut OrdersContainer<Ms, Mdl, INodes, GMs>) -> Init<Mdl> + 'static,
-        update: UpdateFn<Ms, Mdl, INodes, GMs>,
-        view: ViewFn<Mdl, INodes>,
-    ) -> InitAppBuilder<Ms, Mdl, INodes, GMs> {
-        Self::builder(update, view).init(Box::new(init))
-    }
-
-    /// App initialization: Collect its fundamental components, setup, and perform
-    /// an initial render.
-    #[deprecated(
-        since = "0.4.2",
-        note = "Please use `AppBuilder.build_and_start` instead"
-    )]
-    pub fn run(mut self) -> Self {
```

Comment:
This was moved up into `App::start`. Future efforts will want to simplify this, perhaps?